go.mod

```diff
@@ -9,6 +9,7 @@ require (
 	github.com/containerd/continuity v0.0.0-20201208142359-180525291bb7
 	github.com/containerd/go-cni v1.0.1
 	github.com/containerd/stargz-snapshotter/estargz v0.4.1
+	github.com/coreos/go-systemd v0.0.0-20190321100706-95778dfbb74e
```

Comment:
We should use go-systemd/v22 module https://github.com/coreos/go-systemd/releases/tag/v22.1.0

Reply:
`k8s.io/apiserver`  depends on (`95778dfbb74e` = [v19?](https://github.com/coreos/go-systemd/releases/tag/v19)) and `go mod tidy` automatically downgrades to that version. 

https://github.com/kubernetes/apiserver/blob/217e5eef2f6e872b67c1a2a7150152962f341dba/go.mod#L11

Reply:
If you import v22 explicitly it wouldn't cause automatic downgrade

See https://github.com/containerd/containerd/blob/d8527867057daa93c17423ec7cd3438696b1c899/cmd/containerd/command/notify_linux.go#L24

Reply:
Thanks, fixed this.